specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -103,76 +129,120 @@ union ManagedIdentityType {
 @doc("Types of certificates supported.")
 union CertificateType {
   string,
-  @doc("If the certificate is stored in an Azure Key Vault.") "AKV_CERT_URI",
+
+  @doc("If the certificate is stored in an Azure Key Vault.")
+  KeyVaultCertificateUri: "AKV_CERT_URI",
 }
 
 @doc("Types of file supported.")
 union FileType {
   string,
 
   @doc("If the file is a JMX script.")
-  "JMX_FILE",
+  JmxFile: "JMX_FILE",
 
   @doc("If the file is a user properties file.")
-  "USER_PROPERTIES",
+  UserProperties: "USER_PROPERTIES",
 
   @doc("If the file is not among any of the other supported file types.")
-  "ADDITIONAL_ARTIFACTS",
+  AdditionalArtifacts: "ADDITIONAL_ARTIFACTS",
 
   @added(APIVersions.v2023_04_01_preview)
   @doc("If the file is a compressed archive containing a collection of various artifacts or resources.")
-  "ZIPPED_ARTIFACTS",
+  ZippedArtifacts: "ZIPPED_ARTIFACTS",
 
   @added(APIVersions.v2023_04_01_preview)
   @doc("If the file is a JSON config file to define the requests for a URL test.")
-  "URL_TEST_CONFIG",
+  UrlTestConfig: "URL_TEST_CONFIG",
 
   @added(APIVersions.v2024_03_01_preview)
   @doc("If the file is a test script.")
-  "TEST_SCRIPT",
+  TestScript: "TEST_SCRIPT",
 }
 
 @doc("File status.")
-union FileStatus {
+union FileValidationStatus {
   string,
-  @doc("File is not validated.") "NOT_VALIDATED",
-  @doc("File is validated.") "VALIDATION_SUCCESS",
-  @doc("File validation is failed.") "VALIDATION_FAILURE",
-  @doc("File validation is in progress.") "VALIDATION_INITIATED",
-  @doc("Validation is not required.") "VALIDATION_NOT_REQUIRED",
+
+  @doc("File is not validated.")
+  NotValidated: "NOT_VALIDATED",
+
+  @doc("File is validated.")
+  ValidationSuccess: "VALIDATION_SUCCESS",
```

Comment:
should these just be Success, Failed, Initiatied, NotRequired

Reply:
I kept it matching to the original API variants today